src/Symfony/Component/DomCrawler/Form.php

```diff
@@ -44,7 +44,7 @@ class Form extends Link implements \ArrayAccess
      *
```

Comment:
I did not fix the docblock as there are plenty of missing `|null`  across the component codebase, should be done globally on 3.4.